src/commands/document-store/transfer-ownership.ts

```diff
@@ -35,13 +37,33 @@ export const handler = async (args: DocumentStoreTransferOwnershipCommand): Prom
   trace(`Args: ${JSON.stringify(args, null, 2)}`);
   try {
     info(`Transferring ownership to wallet ${args.newOwner}`);
-    const { transactionHash } = await transferDocumentStoreOwnershipToWallet({
+    const account = addAddressPrefix(args.newOwner);
+    const role = "admin";
+    const { transactionHash: grantTransactionHash } = await grantDocumentStoreRole({
       ...args,
       // add 0x automatically in front of the hash if it's not provided
-      newOwner: addAddressPrefix(args.newOwner),
+      account,
+      role,
     });
+
+    const wallet = await getWalletOrSigner({ ...args });
+    const walletAddress = await wallet.getAddress();
+
+    const { transactionHash: revokeTransactionHash } = await revokeDocumentStoreRole({
+      ...args,
+      account: addAddressPrefix(walletAddress),
+      role,
+    });
+
     success(`Ownership of document store ${args.address} has been transferred to new wallet ${args.newOwner}`);
-    info(`Find more details at ${getEtherscanAddress({ network: args.network })}/tx/${transactionHash}`);
+    info(
+      `Find more details at ${getEtherscanAddress({
+        network: args.network,
+      })}/tx/${grantTransactionHash} (grant) and ${getEtherscanAddress({
+        network: args.network,
+      })}/tx/${revokeTransactionHash} (revoke)`
+    );
```

Comment:
Should we send the output as soon as the grant transaction is completed instead of waiting for both to finish then show them together? I'm thinking what if grant was successful but revoke failed? Then the display for the first successful transaction won't be known?